Test

Kaksisilma opened the pull request:
<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You'll find a README and example file in .\html\changelogs\ for further instructions.

You can also find a template for adding your changelog directly to the PR description here: https://github.com/Baystation12/Baystation12/wiki/Automatic-changelog-generation
-->

Spookerton committed:
514 -> 515 change blob 3

AffectedArc07 commented:
Could you please stick `[IDB IGNORE]` and `[MDB IGNORE]` in the PR title please?

CrimsonShrike committed:
Ports weather from nebula

halworsen committed:
refactor overmap movement

move icon pixel movement to base overmap object

use movement indicators for all moving objects

fix overmap subtypes not processing movement

utility procs for making overmap objs movable/stationary

fix heading indicators not showing on sensors

make heading indicators ignore parent color

min speed logic fixes

fix overmap wraparound

fix heading overlay dirs not updating

randomize initial position of overmap objects slightly

remove remaining ship moving_states

increase minimum speed

make_stationary drops speed

get rid of overmap_move_constant

use new global const macro

review

SierraKomodo committed:
Add legion recall procs

SierraKomodo committed:
Door prying now applies to all ai_holders

SierraKomodo committed:
Framework for mobs to "hack" open working doors

SierraKomodo committed:
Legion can hack doors

SierraKomodo committed:
Fix mobs attacking doors while trying to open them

HeyBanditoz committed:
Refactor bottle presets, add most medications

nufc
* Decouples all bottle presets from bottle.dm into their own file
* Bottles with presets get their own preset, most medications and some other chemical included
* Should make it easier to add certain chemicals to maps/loadouts, and adminbus in certain chems, if needed

Spookerton committed:
GLOB list style tweaks

Spookerton committed:
guarantee GLOB.sound_channels is available for other GLOBs that rely on it

Spookerton committed:
GLOB.build_path_to_design_datum_path loose proc to initializer

Spookerton committed:
fix up more globs

GLOB.actual_error_file_line has its type hint
mech GLOBs use the minimum required GLOBAL_ def
GLOB.musical_config uses implicit new

Spookerton committed:
remove /repository/New

ze goggles
just another step in minimizing the signature of /repository before they
become singletons later

Spookerton committed:
icon2base64 can't leak between runs

no longer permits specifying a key, uses a non-persisting savefile
also gets about a 30% performance improvement, so there's that

Detritus-Ha committed:
Pill bottles check for chemmasters

Co-Authored-By: SierraKomodo <sierrakomodo@hey.com>

Bay12Bot committed:
Automatic changelog generation [ci skip]

Jebediahtechnic committed:
Hyposprays can now inject through bags

Hyposprays can now inject through bags + Coding Fixes + Suggestions

Hyposprays can now inject through bags + Indent Fix + Suggestions...
Squashing suggestions...
use_before Fix...
Irrelevant Check...

Proc return Fix

Bag Open + Hypospray Check Adds

TEST CODE

Co-Authored-By: SierraKomodo <sierrakomodo@hey.com>
Co-Authored-By: Hayden <hayden@banditoz.io>
Co-Authored-By: emmanuelbassil <6874235+emmanuelbassil@users.noreply.github.com>

SierraKomodo committed:
Merge pull request #35091 from Spookerton/spkrtn/cng/hardsuit-defib-fix

hardsuit defibs require two hands free

Bay12Bot committed:
Automatic changelog generation for PR #35091 [ci skip]

SierraKomodo committed:
Merge pull request #35100 from Retrokinesis/sestrisite-language-fix

Sestrisite Language Fix

Bay12Bot committed:
Automatic changelog generation [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

SierraKomodo committed:
Fix shield wall gens being unwrenchable

SierraKomodo committed:
Glasses and HUDs now use the same sound effects

Ryan180602 committed:
e14b trinkets

MuckerMayhem committed:
Merge pull request #35104 from SierraKomodo/tweak/hud-glass-sounds

Glasses and HUDs now use the same sound effects

Bay12Bot committed:
Automatic changelog generation for PR #35104 [ci skip]

CrimsonShrike committed:
Generalizes space ambience so that it works as expected with light and ambient updates

Bay12Bot committed:
Automatic changelog generation [ci skip]

SierraKomodo committed:
Merge pull request #35103 from SierraKomodo/bugfix/wrenchable-shield-gen

Fix shield wall gens being unwrenchable

Bay12Bot committed:
Automatic changelog generation for PR #35103 [ci skip]

MuckerMayhem committed:
Merge pull request #35106 from CrimsonShrike/FixShuttleAmbient

Generalizes space ambience so that it works as expected with light an…

MuckerMayhem committed:
Merge pull request #35078 from halworsen/live-laugh-love

Add a wholesome poster to the therapy room

MuckerMayhem committed:
Merge pull request #35099 from Detritus-Ha/dev

Fixes pill bottle interaction with chemmasters

Bay12Bot committed:
Automatic changelog generation for PR #35078 [ci skip]

Bay12Bot committed:
Automatic changelog generation for PR #35099 [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

CakeQ committed:
blood overlay handling rework

SierraKomodo committed:
Merge pull request #33380 from Spookerton/spkrtn/sys/515-3

515-3

Spookerton committed:
add image-rendering css for deprecated ms-interpolation-mode

Spookerton committed:
prefix byond topic links with byond://

various replacements, final discovery with /[='"](?<!byond://)\?\w+=/

Bay12Bot committed:
Automatic changelog generation [ci skip]

HeyBanditoz committed:
Fix AI core wrenching

Spookerton committed:
515 -> 516 version compat

various things not included since they relate to new alist and vector types

Spookerton committed:
decontested new  516 "caller" keyword

Spookerton committed:
reposition renderer on screen for 516

Spookerton committed:
Merge pull request #35115 from HeyBanditoz/ai_core_wrenchdown

Fix AI core wrenching

Bay12Bot committed:
Automatic changelog generation for PR #35115 [ci skip]

Spookerton committed:
Merge pull request #35105 from Ryan180602/legion-trinkets

Add trinkets from Northstar station

Bay12Bot committed:
Automatic changelog generation for PR #35105 [ci skip]

Spookerton committed:
Merge pull request #35113 from Spookerton/spkrtn/sys/516-1

516 compatibility 1

Spookerton committed:
Merge pull request #35116 from Spookerton/spkrtn/cng/globular

mixed global tweaks

Spookerton committed:
default config minimum version to 515.1647

Spookerton committed:
515: fix (most) airlock oddness

Spookerton committed:
Merge pull request #35119 from Spookerton/spkrtn/fix/515-airlock-calls

515: fix (most) airlock oddness

Spookerton committed:
Merge pull request #35117 from Spookerton/spkrtn/sys/515-min-ver

default config minimum version to 515.1647

Bay12Bot committed:
Automatic changelog generation [ci skip]

CrimsonShrike committed:
ZAS and atmos changes, planets and visitable overmap locations now can define external atmos

Ryan180602 committed:
let autopsy scanner only work on dead people

Spookerton committed:
516 display tweaks for browser content

Bay12Bot committed:
Automatic changelog generation [ci skip]

SierraKomodo committed:
Fix empty jobs causing antag runtimes

Spookerton committed:
Merge pull request #35126 from SierraKomodo/bugfix/antag-spawn-runtime

Fix empty jobs causing antag runtimes

Bay12Bot committed:
Automatic changelog generation [ci skip]

Spookerton committed:
515 astar

Spookerton committed:
sort and predicate GLOBAL_PROC_REF

erroneous sortlist -> sorttim in subsystem/mapping

Spookerton committed:
assorted proc_ref, type_proc_ref

BurpleBineapple committed:
SolOp shield cooldown

SierraKomodo committed:
Fix grammar in BP_GROIN targeted attacks

Bay12Bot committed:
Automatic changelog generation [ci skip]

Spookerton committed:
Merge pull request #35131 from BurpleBineapple/nat20

SolOp Shield Cooldown

Bay12Bot committed:
Automatic changelog generation for PR #35131 [ci skip]

Spookerton committed:
Merge pull request #35132 from SierraKomodo/cup-check-grammar-fix

Cup Check Grammar Fix

Bay12Bot committed:
Automatic changelog generation for PR #35132 [ci skip]

Spookerton committed:
misc style tweaks

Spookerton committed:
Merge pull request #35133 from Spookerton/spkrtn/sys/515-4

515 4

Bay12Bot committed:
Automatic changelog generation [ci skip]

HeyBanditoz committed:
Upgraded body scanners check for mutation existence

Requires at least an upgrade to two advanced scanning modules, or one
phasic module.

Spookerton committed:
remove unused params from hotspot_expose

SierraKomodo committed:
Merge pull request #35101 from SierraKomodo/project/legion/recall

Add legion recall procs

Bay12Bot committed:
Automatic changelog generation for PR #35101 [ci skip]

SierraKomodo committed:
Merge pull request #35123 from Ryan180602/autopsy

Let autopsy scanner only work on dead people

Bay12Bot committed:
Automatic changelog generation for PR #35123 [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

Ryan180602 committed:
remove unathi wound/organ regen

MuckerMayhem committed:
fix mutiny antags not spawning correctly

Bay12Bot committed:
Automatic changelog generation [ci skip]

Spookerton committed:
Merge pull request #35141 from MuckerMayhem/fix-mutiny

fix mutiny antags not spawning correctly

Bay12Bot committed:
Automatic changelog generation for PR #35141 [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

MuckerMayhem committed:
Merge pull request #35014 from halworsen/overmap-movement-refactor

Overmap movement refactor

MuckerMayhem committed:
Merge pull request #35036 from SierraKomodo/project/legion/speculator

Legion Speculator

MuckerMayhem committed:
Merge pull request #35023 from SierraKomodo/feature/ai-door-hacking

AI Door Hacking

Bay12Bot committed:
Automatic changelog generation for PR #35014 [ci skip]

Bay12Bot committed:
Automatic changelog generation for PR #35036 [ci skip]

Bay12Bot committed:
Automatic changelog generation for PR #35023 [ci skip]

Spookerton committed:
removed some unused procs & defines

removed cast_new
removed invertHTML
removed InRange
removed AddCamoOverlay
removed BIT_TEST_ALL
removed modulus
removed GaussRand
removed GaussRandRound

HeyBanditoz committed:
Port Nebula's airlock helpers

With adjustments to fix Bay airlocks.
As an alternative for mapping airlocks.

Bay12Bot committed:
Automatic changelog generation [ci skip]

Sweedle committed:
Unproblematics pizza vendor

MuckerMayhem committed:
allow utf8 encoding for paper

halworsen committed:
fix wraparound on north/south overmap edges

Spookerton committed:
Merge pull request #35146 from halworsen/fix-om-wraparound

fix wraparound on north/south overmap edges

Bay12Bot committed:
Automatic changelog generation for PR #35146 [ci skip]

CakeQ committed:
Flasks can be hidden in boots

Bay12Bot committed:
Automatic changelog generation [ci skip]

SierraKomodo committed:
Merge pull request #35144 from Sweedle/dev

SierraKomodo committed:
Trigger example-1 switching to 2-character indexes

Spookerton committed:
Merge pull request #35148 from SierraKomodo/tweak/example-map-double-character-thing

Trigger example-1 switching to 2-character indexes

dependabot[bot] committed:
Bump JamesIves/github-pages-deploy-action from 4.7.2 to 4.7.3

Bumps [JamesIves/github-pages-deploy-action](https://github.com/jamesives/github-pages-deploy-action) from 4.7.2 to 4.7.3.
- [Release notes](https://github.com/jamesives/github-pages-deploy-action/releases)
- [Commits](https://github.com/jamesives/github-pages-deploy-action/compare/15de0f09300eea763baee31dff6c6184995c5f6a...6c2d9db40f9296374acc17b90404b6e8864128c8)

---
updated-dependencies:
- dependency-name: JamesIves/github-pages-deploy-action
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump actions/cache from 4.2.0 to 4.2.1

Bumps [actions/cache](https://github.com/actions/cache) from 4.2.0 to 4.2.1.
- [Release notes](https://github.com/actions/cache/releases)
- [Changelog](https://github.com/actions/cache/blob/main/RELEASES.md)
- [Commits](https://github.com/actions/cache/compare/1bd1e32a3bdc45362d1e726936510720a7c30a57...0c907a75c2c80ebcb7f088228285e798b750cf8f)

---
updated-dependencies:
- dependency-name: actions/cache
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Spookerton committed:
Merge pull request #35150 from Baystation12/dependabot/github_actions/actions/cache-4.2.1

Bump actions/cache from 4.2.0 to 4.2.1

Spookerton committed:
Merge pull request #35149 from Baystation12/dependabot/github_actions/JamesIves/github-pages-deploy-action-4.7.3

Bump JamesIves/github-pages-deploy-action from 4.7.2 to 4.7.3

jwc1015 committed:
Added Bee Plush & Fixed Farwa Plush

Bay12Bot committed:
Automatic changelog generation [ci skip]

SierraKomodo committed:
Increase secure closet minimum damage threshold

SierraKomodo committed:
Pass use_* calls from grab item to grabbed mob

Spookerton committed:
Merge pull request #35151 from Spookerton/spkrtn/cahtoa

remove unused params from hotspot_expose

Spookerton committed:
you can kill pipes to death

Spookerton committed:
Merge pull request #35156 from Spookerton/spkrtn/cng/pipe-fall-down-go-boom

you can kill pipes to death

Bay12Bot committed:
Automatic changelog generation for PR #35156 [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

Spookerton committed:
Merge pull request #35124 from Spookerton/spkrtn/sys/516-2

516 pt 2

Spookerton committed:
add debug->devtools

Spookerton committed:
Merge pull request #35164 from Spookerton/spkrtn/cng/devtools

add debug->devtools

Bay12Bot committed:
Automatic changelog generation for PR #35164 [ci skip]

MuckerMayhem committed:
Merge pull request #35145 from MuckerMayhem/paper-html-fix

allow utf8 encoding for paper

Bay12Bot committed:
Automatic changelog generation for PR #35145 [ci skip]

MuckerMayhem committed:
Merge pull request #35152 from jwc1015/bee

Added Bee Plush & Fixed Farwa Plush

MuckerMayhem committed:
Merge pull request #35034 from CakeQ/filthy-bois

blood overlay handling rework

Bay12Bot committed:
Automatic changelog generation for PR #35034 [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

MuckerMayhem committed:
Merge pull request #35139 from Ryan180602/unathi-heal

Remove unathi wound/organ regen (Keeps their limb regen)

Bay12Bot committed:
Automatic changelog generation for PR #35139 [ci skip]

Ryan180602 committed:
pipes dont die in fire again

MuckerMayhem committed:
remove legion related orders from the ship

Bay12Bot committed:
Automatic changelog generation [ci skip]

MuckerMayhem committed:
Merge pull request #35168 from Ryan180602/pipestuff

Pipes dont die in fire again

Bay12Bot committed:
Automatic changelog generation for PR #35168 [ci skip]

MuckerMayhem committed:
Merge pull request #35153 from SierraKomodo/tweak/secure-closet-min-damage

Increase secure closet minimum damage threshold

Bay12Bot committed:
Automatic changelog generation for PR #35153 [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

Spookerton committed:
Merge pull request #35155 from SierraKomodo/feature/grab-click-passthrough

Pass `use_*` calls from grab item to grabbed mob

Bay12Bot committed:
Automatic changelog generation for PR #35155 [ci skip]

Spookerton committed:
Merge pull request #35169 from MuckerMayhem/remove-legion-envelopes

remove legion related orders from the ship

Bay12Bot committed:
Automatic changelog generation for PR #35169 [ci skip]

Spookerton committed:
add FLAG_N defines, N is 1..24 instead of 0..23

replaced compile time FLAG(N) with FLAG_N
replaced EMPTY_BITFIELD with FLAGS_OFF
added FLAGS_ON as the explicit opposite of FLAGS_OFF
replaced GLOB.bitflags with GLOB.list/index_to_flag
added GLOB.alist/flag_to_index behind 516 preprocessor gate

HeyBanditoz committed:
Pressing "Cancel" in Set Ambition now discards edits

Before, it would just delete your ambitions.

Detritus-Ha committed:
The kind of fancy extractor

Added and loaded up the extractor

+ tiny grammar checks
+ nanoui

Co-Authored-By: Hayden <banditoz@protonmail.com>

Bay12Bot committed:
Automatic changelog generation [ci skip]

Spookerton committed:
refactor SSambient_lighting

FLAG() behavior changed and I got too into it while I was here
should feature improved performance

Spookerton committed:
add Compiler: [DM_VERSION].[DM_BUILD] line to stat

since we're 515/here

Spookerton committed:
rename FLAG() to RFLAG()

this is a deliberate compatibility break to ID any PRs that need adjusting

Spookerton committed:
Merge pull request #35174 from Spookerton/spkrtn/cng/add-compiler-version-mc-stat

add Compiler: [DM_VERSION].[DM_BUILD] line to stat

Bay12Bot committed:
Automatic changelog generation for PR #35174 [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

HeyBanditoz committed:
Update Tracy binaries

Supports up to 515.1647, binaries from Aurorastation

Spookerton committed:
Merge pull request #35177 from HeyBanditoz/tracy_update

Update Tracy binaries

Spookerton committed:
Merge pull request #35173 from Spookerton/spkrtn/cng/flagthulu

FLAG() changes, ssambient_lighting refactor

HeyBanditoz committed:
Medical scanner modules now beep on scan (PDAs, tablets)

Just like health analyzers

dependabot[bot] committed:
Bump actions/cache from 4.2.1 to 4.2.2

Bumps [actions/cache](https://github.com/actions/cache) from 4.2.1 to 4.2.2.
- [Release notes](https://github.com/actions/cache/releases)
- [Changelog](https://github.com/actions/cache/blob/main/RELEASES.md)
- [Commits](https://github.com/actions/cache/compare/0c907a75c2c80ebcb7f088228285e798b750cf8f...d4323d4df104b026a6aa633fdb11d772146be0bf)

---
updated-dependencies:
- dependency-name: actions/cache
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Spookerton committed:
Merge pull request #35181 from Baystation12/dependabot/github_actions/actions/cache-4.2.2

Bump actions/cache from 4.2.1 to 4.2.2

Spookerton committed:
mob/Login only has one override

Spookerton committed:
Merge pull request #35182 from Spookerton/spkrtn/cng/mob-login-squish

mob/Login only has one override

Bay12Bot committed:
Automatic changelog generation [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

CakeQ committed:
Fixes blood transferral on hugs

Spookerton committed:
unshare renderers

Spookerton committed:
Merge pull request #35192 from Spookerton/spkrtn/cng/unshare-renderers

unshare renderers / 516 clients break less

Spookerton committed:
remove some unecessary terminators

Spookerton committed:
replaced dd_range with builtin clamp

Spookerton committed:
move get_teleport_loc adjacent to its only use

Spookerton committed:
clean up sanitize_frequency & format_frequency

Spookerton committed:
moved freeborg and ionnum adjacent to their uses

Spookerton committed:
removed hasvar

Spookerton committed:
move area type procs from unsorted to areas

Spookerton committed:
replace get_areas with get_area_instances, remove get_area_all_atoms

Spookerton committed:
remove oview_or_orange

Spookerton committed:
parse_zone uses a switch

Spookerton committed:
removed global scoped "get" proc

wow

Spookerton committed:
move various item helpers from unsorted to ... item helpers

Spookerton committed:
remove get_turf_or_move

Spookerton committed:
move can_operate adjacent its only use

also tweaked for perf and style

Spookerton committed:
remove format_text

Spookerton committed:
replace gotwallitem with get_wall_item

wow

Spookerton committed:
remove topic_link

Spookerton committed:
pull crash_at into macros, next to crash_with

Spookerton committed:
refactor get_random_colour behavior

bigly of comedy

Spookerton committed:
atom get_light_and_color -> copy_light_and_color

MuckerMayhem committed:
Merge pull request #35052 from HeyBanditoz/bottle_presets

Refactor bottle presets, add most medications

MuckerMayhem committed:
Merge pull request #35102 from Jebediahtechnic/Hypospray-Bag-Injection

Hyposprays can now inject through rescue/stasis bags

MuckerMayhem committed:
Merge pull request #35170 from HeyBanditoz/ambition_ux_changes

Pressing "Cancel" in Set Ambition now discards edits

Bay12Bot committed:
Automatic changelog generation for PR #35170 [ci skip]

HeyBanditoz committed:
Remove accidental bridge inaprovaline bottles

Accidentally left in 1196f55bb3f4f2c16176ae3e1e34eb50e585b9ab

Bay12Bot committed:
Automatic changelog generation [ci skip]

MuckerMayhem committed:
Merge pull request #35194 from HeyBanditoz/inaprovaline_prohibition

Remove accidental bridge inaprovaline bottles

Bay12Bot committed:
Automatic changelog generation [ci skip]

Spookerton committed:
Merge pull request #35193 from Spookerton/spkrtn/cng/lazy-housekeeping

lazy housekeeping

Spookerton committed:
fix area copy to area using source coords instead of target for target turfs

Spookerton committed:
Merge pull request #35191 from CakeQ/bloody-hugs-fix

Fixes blood transferral on hugs

Bay12Bot committed:
Automatic changelog generation for PR #35191 [ci skip]

Spookerton committed:
Merge pull request #35196 from Spookerton/spkrtn/fix/holodeck-area-copy

fix area copy to area using source coords instead of target for target turfs

Bay12Bot committed:
Automatic changelog generation for PR #35196 [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

Detritus-Ha committed:
All unary machines now return_air

Bay12Bot committed:
Automatic changelog generation [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

Detritus-Ha committed:
Binary devices now return air

Bay12Bot committed:
Automatic changelog generation [ci skip]

HeyBanditoz committed:
Expand the playable colony's mess and medbay

* Expands the mess hall and kitchen area.
* The lounge "common area" was removed to make room for kitchen space.
* The medbay is now one tile wider.
* Adds a fridge and organ printer to the medbay.

Bay12Bot committed:
Automatic changelog generation [ci skip]

Ryan180602 committed:
doubles name input on serving bowl food

MuckerMayhem committed:
Merge pull request #35147 from CakeQ/bootflask

Flasks can be hidden in boots

MuckerMayhem committed:
Merge pull request #35203 from Ryan180602/food-unbound

Increase name limit on pen -> food / Doubles name limit on serving bowl

Bay12Bot committed:
Automatic changelog generation for PR #35147 [ci skip]

Bay12Bot committed:
Automatic changelog generation for PR #35203 [ci skip]

Spookerton committed:
Merge pull request #35179 from HeyBanditoz/pda_health_beep_beep

Medical scanner modules now beep on scan (PDAs, tablets)

Bay12Bot committed:
Automatic changelog generation for PR #35179 [ci skip]

Spookerton committed:
Merge pull request #35134 from HeyBanditoz/mutation_body_scanning

Upgraded body scanners check for mutation existence

Bay12Bot committed:
Automatic changelog generation for PR #35134 [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

MuckerMayhem committed:
Merge pull request #35197 from Detritus-Ha/analyze

All unary machines are scannable

MuckerMayhem committed:
Merge pull request #35200 from Detritus-Ha/analyze-binary

All binary machines are scannable

Bay12Bot committed:
Automatic changelog generation for PR #35197 [ci skip]

Bay12Bot committed:
Automatic changelog generation for PR #35200 [ci skip]

MuckerMayhem committed:
Merge pull request #35120 from CrimsonShrike/WeatherAndPlanets

Weather and planet Air

Bay12Bot committed:
Automatic changelog generation for PR #35120 [ci skip]

CrimsonShrike committed:
Ports blurbs from nebula, modifies them a bit

Bay12Bot committed:
Automatic changelog generation [ci skip]

CrimsonShrike committed:
rain sound fixes

MuckerMayhem committed:
actually fix blood hugs

Spookerton committed:
Merge pull request #35206 from MuckerMayhem/really-fix-bloodhugs

actually fix blood hugs

Bay12Bot committed:
Automatic changelog generation for PR #35206 [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

MuckerMayhem committed:
Merge pull request #35171 from Detritus-Ha/extraction-v4

Adds the Gas Extractor

MuckerMayhem committed:
Merge pull request #35205 from CrimsonShrike/HaveYouEverHeardTheRain

rain sound fixes

Bay12Bot committed:
Automatic changelog generation for PR #35171 [ci skip]

MuckerMayhem committed:
Merge pull request #35143 from HeyBanditoz/airlock_helpers

Port Nebula's airlock helpers

Bay12Bot committed:
Automatic changelog generation for PR #35143 [ci skip]

MuckerMayhem committed:
Merge pull request #35204 from CrimsonShrike/RibbonFeatures

Ports blurbs from nebula, modifies them a bit

Bay12Bot committed:
Automatic changelog generation [ci skip]

RuneByt3 committed:
prevents robots from renegade

MuckerMayhem committed:
Merge pull request #35202 from HeyBanditoz/playable_colony_mess_medbay_expansion

Expand the playable colony's mess and medbay

Bay12Bot committed:
Automatic changelog generation for PR #35202 [ci skip]

MuckerMayhem committed:
Merge pull request #35207 from RuneByt3/bugfix

prevents robots from renegade

Bay12Bot committed:
Automatic changelog generation for PR #35207 [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

Kaksisilma commented:
Oh fuck I completely forgot about this. My bad. @AffectedArc07
